json/contestants.json

```diff
@@ -718,6 +718,16 @@
       	  "github" : "https://github.com/shivhek25",
       	  "twitter" : "",
 	        "referral" : ""
+      }  ,
+	    {
+      	  "name" : "Shantam Shrestha",
+      	  "imageurl" : "logan.jfif",
```

Comment:
What this image refers to? Please fix this by adding a image and referring that here.

Reply:
![prob](https://user-images.githubusercontent.com/35340673/34906388-b9a9a578-f892-11e7-9819-d6ce4d99c78c.PNG)

this is because i was not able to upload the proper file

Reply:
Upload to your forked repo not main repo.

Reply:
Sir, I've done as said. I've created a new pull request as add Shantam Shrestha to contestants. I uploaded my branch of forked repo.